### ISSUE TITLE
Prevent TopAppbar from partially visible

### ISSUE THREAD

**Issue by Rahulkr2** [onebone/compose-collapsing-toolbar#34]
If scroll is more than 50% than for positive scroll show toolbar and negative scroll hide toolbar.
I don't want toolbar to be partially visible.
Please tell me how to achieve this behavior.

**Comment by onebone** [comment]
There is no official way to implement this, but is being implemented with #14.

For now, you can implement it by intercepting `nestedScroll` by yourself.
```kotlin
CollapsingToolbarScaffold(
	modifier = Modifier.fillMaxSize()
		.nestedScroll(object : NestedScrollConnection {
			override suspend fun onPostFling(
				consumed: Velocity,
				available: Velocity
			): Velocity {
				@OptIn(ExperimentalToolbarApi::class)
				if (state.toolbarState.progress < 0.5) {
					state.toolbarState.collapse()
				} else {
					state.toolbarState.expand()
				}

				return Velocity.Zero
			}
		}),
	// ...
)
```

**Comment by SECURISTAX** [comment]
[]()
